pages/parent/enrollment.tsx

```diff
@@ -77,6 +80,12 @@ export default function ParentEnrollClass({
         <MediaReleaseForm onNext={nextPage} />,
         <ParticipantWaiver onNext={nextPage} />,
         <TermsAndConditions onNext={nextPage} />,
+        <ProofOfIncomeWrapper
```

Comment:
Instead of passing a wrapper here, we can use a ternary on the high level To see if need the proof of income page, discount page, or nothing.

Kinda like what Brandon did here: https://github.com/uwblueprint/social-diversity-for-children/pull/107/files#diff-f65ce9fe6779f3e25edf7295e71921c959fa8fe6bb286360b0868c27c3a9658aR81-R109